calabash-cucumber/lib/calabash-cucumber/launch/simulator_launcher.rb

```diff
@@ -251,13 +251,24 @@ def app_bundle_or_raise(path=nil, device_build_dir='iPhoneSimulator')
             puts('-'*37)
           end
         else
-          bo_dir = build_output_dir_for_project
-          sim_dirs = Dir.glob(File.join(bo_dir, '*-iphonesimulator', '*.app'))
+          search_dir = build_output_dir_for_project || DERIVED_DATA
+          sim_dirs = ''
+          apps = `find #{search_dir} -name "*.app" | sort -n`.split("\n")
+          apps.each do |app_path|
+            if app_path.split("/")[-2].include? 'iphonesimulator'
```

Comment:
The would be better as:

```
if app_path[/iphonesimulator/, 0]
```

It is also possible to have path _without_ "-iphonesimulator" - we've seen some evidence of this in Xcode 7.2.  A better check would be to ask the App what its arches are.

```
lipo = RunLoop::Lipo.new(app_path)
arches = lipo.info

if arches.include?("x86_64") || arches.include?("i386")
   # simulator build
else
  # arm build
end
```